index.html

```diff
@@ -554,16 +554,23 @@ <h2>
 &lt;/button&gt;
 </pre>
       <p>
-        This example waits to be fullscreen before locking the screen
-        orientation and starting.
+        This example waits to be fullscreen and locked to landscape before
+        running <code>ready()</code>.
       </p>
       <pre class='example html'>
 &lt;script&gt;
-  var start = function() {
-    document.onfullscreenchange = function() {
-      screen.orientation.lock('natural').then(startInternal);
-    }
-    document.documentElement.requestFullscreen();
+  function ready(){
+    console.log("Now full screen and ready to start, locked to " + screen.orientation.type);
+  }
+
+  async function start() {
+    const promiseToFullScreen = new Promise(resolve =&gt;
```

Comment:
As we discussed, we don't need this anymore :) Similarly if it's still in other examples.